src/CMakeLists.txt

```diff
@@ -6,13 +6,13 @@ endif()
 
 add_definitions(-DHYRISE_DEBUG=${HYRISE_DEBUG})
 
-# Provide ENABLE_JIT_SUPPORT option and automatically disable JIT if compiler is not Clang
-# ENABLE_JIT_SUPPORT is OFF on MacOS by default as JIT does not work with -O2 and -O3 on MacOS
-CMAKE_DEPENDENT_OPTION(ENABLE_JIT_SUPPORT "Build with JIT support" OFF APPLE ON)
-if (NOT ${LLVM_FOUND} OR NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.0)
+# Provide ENABLE_JIT_SUPPORT option and automatically disable JIT if compiler is not Clang 6.*
+# ENABLE_JIT_SUPPORT is OFF on MacOS by default as it currently does not work on MacOS
```

Comment:
Could you give a one line summary on why it doesn't?

Reply:
I added a comment and referenced the new issue #1569